handler/src/main/java/io/netty/handler/ssl/OpenSslServerContext.java

```diff
@@ -439,6 +439,7 @@ void verify(OpenSslEngine engine, X509Certificate[] peerCerts, String auth)
                 }
             }
             sessionContext = new OpenSslServerSessionContext(ctx);
+            sessionContext.setSessionIdContext("netty".getBytes());
```

Comment:
Please define a static byte[] and reuse everytime. Something like:

private static final byte[] ID = new byte[] {'n', 'e', 't', 't', 'y'};